zeebe-operate/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta1
```

Comment:
@PashCracken can you do the same check here?

Reply:
@salaboy I’ll add the appropriate check on Monday

Reply:
@salaboy Deployment apps/v1 has been available since Kubernetes version 1.9. Do you really want to support versions older than that? If not, these changes will work fine for version 1.9 and newer

Reply:
@salaboy Deployment apps/v1 has been available since Kubernetes version 1.9. Do you really want to support versions older than that? If not, these changes will work fine for version 1.9 and newer

Reply:
@PashCracken.. you are right.. I was checking that exactly.. I don't think that there is need for that.. I will merge and test the new versions with 1.14